pydrive2/test/settings/test1.yaml

```diff
@@ -1,9 +1,12 @@
-client_config_backend: 'file'
-client_config_file: 'configs/client_secrets.json'
+client_config_backend: file
```

Comment:
I still think we should not be changing test1.yaml ... it was made to test a different workflow